Framework/DataHandling/src/DetermineChunking.cpp

```diff
@@ -125,24 +120,9 @@ void DetermineChunking::exec() {
   }
   this->setProperty("OutputWorkspace", strategy);
 
-#ifndef MPI_BUILD
-  // mpi needs work for every core, so don't do this
```

Comment:
Should this block of code be kept because it is in `ifndef`?

Reply:
Hmm, I'm not sure. It does set the output property from the empty workspace so maybe there's a reason that was done. I think it's safer to leave it where it is for now.